src/ape/api/providers.py

```diff
@@ -716,7 +716,11 @@ def send_call(self, txn: TransactionAPI, **kwargs) -> bytes:
             raise self.get_virtual_machine_error(err) from err
 
     def get_transaction(
-        self, txn_hash: str, required_confirmations: int = 0, timeout: Optional[int] = None
+        self,
+        txn_hash: str,
+        required_confirmations: int = 0,
+        timeout: Optional[int] = None,
+        raise_on_fail: bool = True,
```

Comment:
Should it be `False` here so we can get failed receipts easily and then have it get set to `True` during contract invocations / making transactions?

Reply:
@unparalleled-js talked with Bryant on Friday and we settled on some slightly different behavior, I set the value to true in 6f79956 and added an explanation at https://github.com/ApeWorX/ape/pull/957#issuecomment-1215565267